xbmc/video/dialogs/GUIDialogAudioSubtitleSettings.cpp

```diff
@@ -293,9 +293,13 @@ void CGUIDialogAudioSubtitleSettings::InitializeSettings()
     g_application.m_pPlayer->GetSubtitleCapabilities(m_subCaps);
   }
 
+  // register IsPlayingPassthrough condition
+  m_settingsManager->AddCondition("IsPlayingPassthrough", IsPlayingPassthrough);
+
   CSettingDependency dependencyAudioOutputPassthroughDisabled(SettingDependencyTypeEnable, m_settingsManager);
-  dependencyAudioOutputPassthroughDisabled.And()
-    ->Add(CSettingDependencyConditionPtr(new CSettingDependencyCondition(SETTING_AUDIO_PASSTHROUGH, "false", SettingDependencyOperatorEquals, false, m_settingsManager)));
+  dependencyAudioOutputPassthroughDisabled.Or()
```

Comment:
Just checked and it basically did the same thing, see https://github.com/xbmc/xbmc/blob/4d19be5bc321d68973058a9b9df8d4aacae89892/xbmc/video/dialogs/GUIDialogAudioSubtitleSettings.cpp#L286. Is it possible that this didn't really work in Gotham either? Or it probably worked fine when opening the dialog but changing any of the other settings would also have resulted in an invalid state.